client/my-sites/plans-features-main/wpcom-faq.jsx

```diff
@@ -142,8 +142,7 @@ const WpcomFAQ = ( { isChatAvailable, siteSlug, translate } ) => {
 			<FAQItem
 				question={ translate( 'Can I cancel my subscription?' ) }
 				answer={ translate(
-					'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day' +
-						' refund on all of our plans. {{a}}Manage purchases{{/a}}.',
+					'Yes. We want you to love everything you do at WordPress.com, so we provide a 14-day refund on all of our annual plans and a 7-day refund on all of our monthly plans. {{a}}Manage purchases{{/a}}.',
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **27** times:
`translate( 'Yes. We want you to love everything you do at WordPress.com, so we provide a 30-day refund on all of our plans. {{a}}Manage purchases{{/a}}.' )` *ES Score: 6*